test/components/selectionBoxLayerTests.ts

```diff
@@ -1,193 +1,244 @@
 ///<reference path="../testReference.ts" />
 
-describe("SelectionBoxLayer", () => {
-  it("boxVisible()", () => {
-    let svg = TestMethods.generateSVG();
-    let sbl = new Plottable.Components.SelectionBoxLayer();
-    sbl.renderTo(svg);
+describe("Interactive Components", () => {
+  describe("SelectionBoxLayer", () => {
+
+    describe("Basic Usage", () => {
+      let svgWidth = 500;
+      let svgHeight = 500;
+
+      let svg: d3.Selection<void>;
+      let sbl: Plottable.Components.SelectionBoxLayer;
+
+      beforeEach(() => {
+        svg = TestMethods.generateSVG(svgWidth, svgHeight);
+        sbl = new Plottable.Components.SelectionBoxLayer();
+      });
+
+      it("can get and set the boxVisible() property", () => {
+        assert.strictEqual(sbl.boxVisible(), false, "The box is not visible by default");
+
+        sbl.renderTo(svg);
+        assert.strictEqual(sbl.boxVisible(), false, "The box is not visible by default after rendering to svg");
+
+        assert.strictEqual(sbl.boxVisible(true), sbl, "Setting the boxVisible attribute returns the selection box layer");
+        assert.strictEqual(sbl.boxVisible(), true, "Setting the boxVisible to true attribute works");
+        sbl.boxVisible(false);
+        assert.strictEqual(sbl.boxVisible(), false, "Setting the boxVisible attribute to false works");
+
+        svg.remove();
+      });
```

Comment:
As @jtlan was saying. We have to build up our tests on axioms. This tests proves that the getter and the setter are having the expected behavior, so we can use them later assuming they are correct. 

I must agree that the usefulness of this test is below the average though.